script/Ciel/Day 12/12_01_CIEL12_3_1.txt

```diff
@@ -261,7 +276,7 @@ I guess, that's what I love about her, even in the midst of this extreme torment
 [sha:d4186e3cc3ba4810e63afc9afab431eb7d7e75e2]{
 -- Page 169, Offset 31315.
 -- ……ああ、吐き気がする。
-...Ah, I feel like throwing up.
+Ah, I feel like I'm going to be sick...
 }
 [sha:d87368dacdb01ab04452e78b1370ea5b1463a763]{
 -- Page 169, Offset 31316.
```

Comment:
Maybe an 'ugh' is more appropriate than an 'ah'.
That and I do think that coming across a bit stronger here is good. He's clearly already fucked, so a more vivid picture of his state is better.

>Ugh... I feel like I'm about to throw up.